examples/Makefile.am

```diff
@@ -3,7 +3,9 @@ MAINTAINERCLEANFILES = Makefile.in
 noinst_PROGRAMS = list_u32 map_struct
 
 list_u32_SOURCES = list_u32.c
-list_u32_LDADD = ../src/libcdata.la
+list_u32_CPPFLAGS = -I$(top_srcdir)/include/ -D__CDATA_INTERNAL_INCLUDE
+list_u32_LDADD = $(top_srcdir)/src/libcdata.la
```

Comment:
This was OK to be `../src/` else, it will break out of tree builds

Includes should be fine. I will take your commit and amend it.